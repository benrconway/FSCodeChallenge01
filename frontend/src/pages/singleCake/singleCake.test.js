import React from "react";
import { render } from "@testing-library/react";
import SingleCake from "./";

test("renders learn react link", () => {
  const { getByText } = render(<SingleCake />);
  const linkElement = getByText(/SingleCake screen/i);
  expect(linkElement).toBeInTheDocument();
});
