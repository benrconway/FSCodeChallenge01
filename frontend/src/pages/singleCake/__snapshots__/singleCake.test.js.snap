// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders single cake screen 1`] = `
<DocumentFragment>
  <div>
    <button>
      &lt; Back
    </button>
    <div
      class="cake-tile"
    >
      <img
        alt="__TEST_CAKE__"
        class="cake-tile-image"
        src="__TEST_URL__"
      />
      <p>
        __TEST_CAKE__
      </p>
      <p>
        Yum Factor: 5
      </p>
      <p>
        __TEST_COMMENT__
      </p>
    </div>
  </div>
</DocumentFragment>
`;
